root/layout/header/bottom.tt

```diff
@@ -152,6 +158,9 @@
             <li>
                 <a href="[% doc_link('Frequently_Asked_Questions') %]">[% l('FAQs') %]</a>
             </li>
+            <li>
+                <a href="[% doc_link('MusicBrainz_Documentation') %]">[% l('Documentation Index') %]</a>
+            </li>
```

Comment:
Another case of "not sure what the best name would be" - this seemed as good as any I guess. This page might not be very useful for a newcomer, not sure, but at least it has a lot of links so that's good, I guess.